DXRFixes/DeusEx/Classes/ScriptedPawn.uc

```diff
@@ -663,6 +692,33 @@ state Sitting
 
 }
 
+// returns true if the damage type can knock the pawn out
+function bool CanKnockUnconscious(coerce string damageType)
```

Comment:
make this a static function inside DXREventsBase so that we can keep the gameplay logic and bingo logic in sync